import "serverinfo_card.proto";

message ServerInfo_Zone {
	enum ZoneType {
		PrivateZone = 0;
		PublicZone = 1;
		HiddenZone = 2;
	}
	optional string name = 1;
	optional ZoneType type = 2;
	optional bool with_coords = 3;
	optional sint32 card_count = 4;
	repeated ServerInfo_Card card_list = 5;
}
